src/css/Title.scss

```diff
@@ -1,117 +1,104 @@
-.title {
-  position: relative;
-  height: 36px;
-  width: 100%;
-
-  .justify-center {
-    justify-content: center;
-  }
-
-  > div > span,
-  div#lower-row > div {
-    width: inherit;
-    font-size: 13px;
-  }
-
-  div#lower-row > div.title-attempts {
-    width: 40px;
-    text-align: right;
-    vertical-align: bottom;
-  }
+@import "variables";
 
-  div#lower-row > div.title-category {
-    width: calc(100% - 80px);
-    padding-left: 40px;
-    display: inline-flex;
-    align-items: center;
-    height: 100%;
-  }
-
-  > div.meta-left > span,
-  div.meta-left > div#lower-row > div:not(.title-attempts) {
-    text-align: left;
-  }
-
-  div.meta-left > div#lower-row > div.title-category {
-    width: calc(100% - 40px);
-    padding: 0;
-    justify-content: baseline;
-  }
-
-  div.title-attempts {
-    position: absolute;
-    right: 6px;
-    bottom: 4px;
-  }
-  div.meta-two-lines > div#lower-row > div.title-category {
-    align-items: baseline;
-    vertical-align: top;
-    padding-top: 1px;
-  }
-
-  div.meta-two-lines > div#lower-row > div.title-attempts {
-    vertical-align: top;
-    padding-bottom: 0;
-    padding-top: 1px;
-  }
+$title-font-size: 13px;
+$attempts-width: 40px;
```

Comment:
Does this need to be a fixed width? Can't we do it like the split times and co. where we dynamically cut off the split names?

Reply:
I guess this may cause problems centering the text then as this value seems to be hardcoded into the centering logic as well.

Reply:
This is harder than the split names because we're trying to center the category text relative to the entire component rather than within just the space that isn't part of the attempt count. I'll try to figure something out though.